samples/perf_hgemm.cpp

```diff
@@ -223,7 +223,48 @@ using namespace rocwmma;
 *|_________|________________________________|
 */
 
-// Types
+namespace gfx9Params
```

Comment:
Request to adjust the wording of the documentation on line 199-202.
* Instead of "maximum performance" can be more general as "high performance". Don't want to assume this will give max perf.
* Can probably remove the part that says "This performance sample is setup for GFX11..." because now both gfx9 and gfx11 configs are supported at the same time.:)